### PR TITLE
Save and return with v2 auth

### PR DESCRIPTION
**Pull request by sequencemedialimited**
- Tweak

### CODE REVIEW

File: app/controllers/email_signins_controller.rb [hunk 1]
```diff
@@ -2,23 +2,31 @@ class EmailSigninsController < ApplicationController
...
+      service_slug: params[:service_slug],
+      encrypted_email: params[:encrypted_email],
+      expires_at: expires_at
+    )
 
     if magic_link.save
-      render json: { token: magic_link.id }, status: :created
+      return render json: { token: magic_link.id }, status: :created
+    else
+      return render_email_missing_error
```
Comment (sequencemedialimited) [inline app/controllers/email_signins_controller.rb]: Added

**Comment by sequencemedialimited** [comment]
Checks fail on coverage not on code

File: app/controllers/email_signins_controller.rb (hunk 1 above)
Comment (brenetic) [reply app/controllers/email_signins_controller.rb]: Are we catching the fact that either the service slug or the encrypted email might be missing from the payload?

Comment (sequencemedialimited) [reply app/controllers/email_signins_controller.rb]: Nope. Not here, anyhoo

Let's look at that next week

**Comment by sequencemedialimited** [comment]
I've added a couple of specs and a test for the `encrypted_email` and `encrypted_details` params @brenetic

**Pull request merged**